ifrs17/Utils/Queries.ipynb

```diff
@@ -183,18 +183,18 @@
         {
             "cell_type": "code",
             "source": [
-                "public static async Task<Dictionary<string, YieldCurve>> LoadLockedInYieldCurveAsync(this IQuerySource querySource, Args args, ",
-                "\n                                                                                                       IEnumerable<DataNodeData> dataNodes)",
+                "public static async Task<Dictionary<string, YieldCurve>> LoadLockedInYieldCurveAsync(this IQuerySource querySource, Args args, IEnumerable<DataNodeData> dataNodes)",
+                "\n",
                 "\n{",
                 "\n    var lockedInYieldCurveByGoc = new Dictionary<string, YieldCurve>();",
                 "\n    foreach (var dn in dataNodes.Where(x => x.ValuationApproach == ValuationApproaches.BBA))",
```

Comment:
I think also this needs improvement.
We don't need this query so many times.

Reply:
If we need to update the args, then we can not optimize it, I think

Reply:
the optimization of the locked query will be done in a following PR